cli/command_snapshot_create.go

```diff
@@ -181,6 +181,10 @@ func getLocalBackupPaths(ctx context.Context, rep *repo.Repository) ([]string, e
 }
 
 func getUserName() string {
+	if len(overrideUserName) > 0 {
+		return overrideUserName
+	}
+
```

Comment:
You may simply want to:
* simply return the `username` variable
* have the previous functions return the default user / host names
* initialize the variables with the default user and host names

So, for example:
```suggestion
	return userName
}

func getDefaultUserName() string {
```